source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -456,6 +485,22 @@ export default class NetworkStatusStore extends Store {
       const networkStatus: GetNetworkInfoResponse = await this.getNetworkInfoRequest.execute()
         .promise;
 
+      const { tempCurrentEpoch, tempNextEpochStart } = this;
+
+      // @DECENTRALIZATION TODO: Remove
+      if (tempCurrentEpoch) {
+        set(networkStatus, 'localTip.epoch', tempCurrentEpoch);
+        set(networkStatus, 'networkTip.epoch', tempCurrentEpoch);
+        set(networkStatus, 'nextEpoch.epochNumber', tempCurrentEpoch + 1);
+      }
+      // $FlowFixMe
+      if (
+        tempNextEpochStart &&
+        !`${new Date(tempNextEpochStart)}`.includes('Invalid')
```

Comment:
@daniloprates Can you isolate this to e.g. `isInvalid`?

Reply:
@tomislavhoracek this is for testing only and the `// $FlowFixMe` will be removed alongside with it

Reply:
👍